### PR TITLE
GraphQLClient plugin

### PR DESCRIPTION
**Pull request by VojtaSim**
Created graphql-client plugin that provides a way to easily make HTTP request to GraphQL endpoint
Plugin contains a loader for webpack implementations and uses http-client plugin to make http requests.

### CODE REVIEW

File: packages/plugin-graphql-client/src/__tests__/indexSpec.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        "setProps": [Function],
+        "setState": [Function],
+        "setup": [Function],
+        "state": Object {},
+        "unmount": [Function],
+        "update": [Function],
+        "version": "1.0.0",
+      }
+    `);
+  });
```
Comment (mjancarik) [inline packages/plugin-graphql-client/src/__tests__/indexSpec.js:125]: add some scenarios for graphql.request method

File: docs/docs/graphql-client-plugin.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+layout: docs
+title: GraphQL client plugin - Merkur
+---
+
+# GraphQL client plugin
+
+The GraphQL client plugin adds `graphql` property to your widget with a `request` method. Under the hood this plugin uses [HTTP Client plugin](https://merkur.js.org/docs/http-client-plugin)
+
+## Installation
+
```
Comment (mjancarik) [inline docs/docs/graphql-client-plugin.md:11]: Missing part: User must add graphql loader to webpack config.

File: packages/plugin-graphql-client/src/__tests__/indexSpec.js (hunk 1 above)
Comment (mrazekD) [reply packages/plugin-graphql-client/src/__tests__/indexSpec.js:164]: Added

File: docs/docs/graphql-client-plugin.md (hunk 2 above)
Comment (mrazekD) [reply docs/docs/graphql-client-plugin.md:11]: Added

File: packages/plugin-graphql-client/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Merkur - plugin-graphql-client
+
```
Comment (mjancarik) [inline packages/plugin-graphql-client/README.md:2]: Change @merkur/plugin-http-client to @merkur/plugin-graphql-client

File: packages/plugin-graphql-client/loader.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function applyGraphqlLoader(config) {
+  config.module.rules.push({
+    test: /\.(graphql|gql)$/,
+    exclude: /node_modules/,
+    loader: 'graphql-tag/loader',
```
Comment (mjancarik) [inline packages/plugin-graphql-client/loader.js:5]: missing dependencies in package.json

Comment (mrazekD) [reply packages/plugin-graphql-client/loader.js:5]: `graphql-tag` is already in `package.json`

**Pull request merged**